README.md

```diff
@@ -368,7 +368,7 @@ ok = hackney_pool:start_pool(PoolName, Options),
 `timeout` is the time we keep the connection alive in the pool,
 `max_connections` is the number of connections maintained in the pool. Each
 connection in a pool is monitored and closed connections are removed
-automatically.
+automatically. The default timeout is `150000` and the default number of connections is `50`.
```

Comment:
Can you add the unit to the timeout? ie.   150000 ms  sot it would definitely clarify the text